.eslintrc.js

```diff
@@ -29,7 +29,11 @@ module.exports = {
 
     'import/no-extraneous-dependencies': ['error', { devDependencies: [
       // only allow devDependencies in these folders:
-      '*.js', '**/__tests__/*', 'frontend/**/*', 'jest/**/*'
+      '*.js',
+      '**/__tests__/*',
+      'jest/**/*',
+      '**/*.stories.jsx',
```

Comment:
@gnarf identified that we had a duplicative `.eslintrc.js` file inside frontend that was defining an override for this rule specifically, preventing the parent config from taking effect inside `frontend/`. Since the intent of the nested file was already accomplished by the existing config in the parent, I've removed the nested `.eslintrc.js`.